src/jira/client/jira-client.ts

```diff
@@ -163,16 +163,24 @@ export const getJiraClient = async (
 						}
 					}),
 				addForIssue: (issue_id: string, payload) =>
-					instance.post("/rest/api/latest/issue/{issue_id}/comment", payload, {
+					instance.post("/rest/api/3/issue/{issue_id}/comment", payload, {
```

Comment:
`latest` doesn't allow for passing ADF, which is what makes the comment "pretty"